assets/src/carousel/index.js

```diff
@@ -135,27 +124,60 @@ domReady(() => {
   };
 
   Array.from(carouselWrappers).forEach((carouselWrapper) => {
-    /* eslint-disable-next-line no-new -- we do not store the object as no further computation required with the built object. */
-    new Glider(carouselWrapper, {
-      // Mobile-first defaults
-      slidesToShow: 1,
-      slidesToScroll: 1,
-      scrollLock: true,
-      arrows: !isRTL ? navArrows : navArrowsRTL,
-      responsive: [
-        {
-          // screens greater than >= 775px
-          breakpoint: 775,
-          settings: {
-            // Set to `auto` and provide item width to adjust to viewport
-            slidesToShow: 'auto',
-            slidesToScroll: 'auto',
-            itemWidth: carouselWrapper.querySelector('.web-stories-list__story')
-              .offsetWidth,
-            duration: 0.25,
+    // For multiple instance of the glider we need to link nav arrows appropriately.
+    const carouselId = carouselWrapper.dataset.id;
+
+    const navArrows = {
+      prev: `.${carouselId} .glider-prev`,
+      next: `.${carouselId} .glider-next`,
+    };
+
+    // This is to add basic support for the RTL. We switch the nav arrows.
+    const navArrowsRTL = {
+      prev: `.${carouselId} .glider-next`,
+      next: `.${carouselId} .glider-prev`,
+    };
```

Comment:
```suggestion
    const navArrows = {
      prev: !isRTL ? `.${carouselId} .glider-prev` : `.${carouselId} .glider-next`,
      next: !isRTL ? `.${carouselId} .glider-next` : `.${carouselId} .glider-prev`,
    };
```

Reply:
Some of the changes in this PR were related to bug fixes so have moved them to #279. Also included this feedback in the same [PR file diff](https://github.com/rtCamp/web-stories-wp/pull/279/files#diff-78514a80d1420cf1df25273c2dcc4c0bd7d0d362b355bb61f3701debc0bad3e0R130).